test/views/AssociateDucrConfirmationViewSpec.scala

```diff
@@ -65,7 +66,10 @@ class AssociateDucrConfirmationViewSpec extends UnitViewSpec with CommonMessages
 
     "display 'Back to start page' button on page" in {
 
-      view.getElementsByClass("button").text() mustBe messages(backToStartPageCaption)
+      val button = view.getElementsByClass("button")
+
+      button.text() mustBe messages(backToStartPageCaption)
+      button.attr("href") must equal(routes.StartController.displayStartPage().url)
```

Comment:
It's better to use our defined matchers ;)
https://github.com/hmrc/customs-movements-frontend/blob/6244e1ff135cdc620f2f1473e0b87b0bb2caddfa/test/views/spec/ViewMatchers.scala#L216

Reply:
Yup, I fixed this